backend/app/views/spree/admin/shared/_navigation.html.erb

```diff
@@ -1,3 +1,4 @@
+<span id="admin-nav-toggle"><i class="fa fa-chevron-left fa-2x"></i></span>
```

Comment:
Instead of changing the innerHTML of the toggle icon in JS we could also use some CSS magic to rotate it if we toggle the body class.